### PR TITLE
Update static error pages to use Zendesk support email (SCP-2643)

### PR DESCRIPTION
**Pull request by jlchang**
Updating static error pages to contact SCP support using "scp-support@broadinstitute.zendesk.com", consolidating all SCP support issues to use Zendesk.

This supports SCP-2643 for this Github repo - there is a parallel PR in our public-facing single_cell_portal repo.

Unfortunately my IDE decided to do some reformatting so it's hard to pick out the changes. There are only three non-formatting differences:

1. change link in page body from GitHub issues to "scp-support@broadinstitute.zendesk.com"
1. fix help dropdown so that they work again (`class=dropdown-toggle"` missing a quotation mark)
1. fix 422.html to use all.css

### CODE REVIEW

File: public/422.html [hunk 1]
```diff
@@ -32,9 +39,10 @@
       background-color: #efefef;
       padding-top: 25px;
       position: relative;
-      margin-top: -155px; /* negative value of footer height */
+      margin-top: -155px;
+      /* negative value of footer height */
       height: 155px;
```
Comment (eweitz) [inline public/422.html:44]: Auto-formatting caused misleading comment placement.  The previous placement seems best:

```suggestion
      margin-top: -155px; /* negative value of footer height */
      height: 155px;
```

File: public/500.html [hunk 2]
```diff
@@ -67,55 +78,65 @@
...
+        <p class="text-muted">&copy; 2017 The Broad Institute of MIT & Harvard</p>
```
Comment (eweitz) [inline public/500.html:136]: Correct year and [preferred form](https://drive.google.com/file/d/0B877YjdN2Vv9ZkdPNElpSTZmTDg/view) (see page 7) for institute name :

```suggestion
        <p class="text-muted">&copy; 2020 Broad Institute of MIT and Harvard</p>
```

File: public/404.html [hunk 3]
```diff
@@ -62,59 +73,72 @@
...
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav pull-right">
+        <li class='dropdown'><a href="#" class="dropdown-toggle" data-toggle="dropdown"><span
+              class='fas fa-question-circle'></span> Help <span class='caret'></span></a>
+          <ul class="dropdown-menu dropdown-menu-right">
+            <li><a class='check-upload' href="https://github.com/broadinstitute/single_cell_portal/wiki"
+                target='_blank'><span class='fas fa-info-circle fa-fw'></span> Documentation/Wiki</a></li>
+            <li><a href="https://github.com/broadinstitute/single_cell_portal/issues" target="_blank"><span
+                  class="fas fa-fw fa-exclamation-circle"></span> Issues</a></li>
+            <li><a href="https://github.com/broadinstitute/single_cell_portal_core" target="_blank"><span
```
Comment (bistline) [inline public/404.html:110]: We still have the link to the issues in the dropdown for all of these files - probably we should change this to just be a direct `mailto` link that says 'Contact Us':

```
<li><a href="mailto:scp-support@broadinstitute.zendesk.com" target="_blank"><span
                  class="fas fa-fw fa-envelope"></span> Contact Us</a></li>
```

Comment (jlchang) [reply public/404.html:110]: updated in `d53230e`

**Pull request merged**